pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -427,6 +432,28 @@ func (p *ConfigurationEditFlags) Apply(
 		servinglib.UpdateUser(template, p.PodSpecFlags.User)
 	}
 
+	if cmd.Flags().Changed("scale-init") {
+		contains := func(s []string, e string) bool {
+			for _, a := range s {
+				if strings.Contains(a, e) {
```

Comment:
Can we better names than `a`, `s`, `e`, etc? :)

Reply:
Yes, I can definitely name it better but it was only a scope limited helper function I wrote that cannot be used outside this block so thought naming won't be necessary.